now.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
it's ok. that's why we should remove firebase as this project is deployed to now service.

Reply:
ok.